apps/aesophia_http/src/aesophia_cors_middleware.erl

```diff
@@ -63,7 +63,8 @@ set_cors_headers(Req, Origin) ->
 cors_headers(Req, Origin) ->
     MaxAge       = ?ACCESS_CONTROL_MAX_AGE,
     CorsHeaders =
-        [{<<"access-control-allow-origin">>      , Origin},
+        [{<<"vary">>                             , <<"origin">>},
```

Comment:
So, the header name handling is supposed to be case-insensitive, but presumably, [software that departs from this](https://support.fastly.com/hc/en-us/community/posts/360040448012-Headers-case-insensitivity-Vary-Accept-Encoding-) is likely to go with all-lowercase, so this is probably perfect.